### ISSUE TITLE
Set up each Phase 1 repo for audit tracking

### ISSUE THREAD

**Issue by erinchambers** [CDRH/accessibility#9]
Phase 1 = ready to audit, no input or coordination needed

Steps for each repo that are a YES on our [Checklist](https://uofnebraska.sharepoint.com/:x:/r/sites/UNL-UniversityLibraries/DISC/cdrh/Shared%20Documents/CDRH%20Projects/Project%20Folders/Accessibility%20Audits/2025-2026%20ADA%20Title%20II%20Project/Planning%20Documents/Checklist%20of%20Products%20for%20ADA%20Title%20II%20Project.xlsx?d=w52aafefaa1a841b09bae916225c22cbb&csf=1&web=1&e=9tXWgN).

# Checklist for each product repo

- [x] Create milestone
- [x] Create labels
  - [x] `a11y`
  - [x] `a11y improvments`
  - [x] `Priority 1: Critical`
  - [x] `Priority 2: Serious`
  - [x] `Priority 3: Moderate`
- [x] Add **Accessibility Audit** issue
  - Assignee: `erinchambers`
  - Label: `a11y`
  - Type: `Task`
  - Projects: `2025 ADA Title II Project`
  - Milestone: `{Product shortname} Accessibility Remediation`
- [x] Add **Accessibility Fixes** task
  - Assignee: `erinchambers`
  - Label: `a11y`
  - Type: `Task`
  - Projects: `2025 ADA Title II Project`
  - Milestone: `{Product shortname} Accessibility Remediation`

# Steps for each product repo

## Step 1: Create milestone

Title: Product shortname, followed by:

```
 Accessibility Remediation
```

Description:

```
Accessibility audit and remediation for the April 24, 2026 ADA Title II compliance deadline.
```

Due Date: April 24, 2026

### a11y
  - dark blue label matching WCAG branding
  - does not indicate severity/priority of issue, only compliance

Description:

```
Accessibility changes REQUIRED to meet ADA Title II requirements (WCAG 2.1 Level AA).
```

Color: 
```
#005a9c
```

### a11y improvements
- light blue label
- issues with this label are NOT required by ADA Title II

Description:

```
Ideas and suggestions for accessibility improvements NOT REQUIRED by WCAG 2.1 Level AA guidelines.
```

```
#95C0E6
```

### Priority 1: Critical 
Color: default bright red

Description:

```
Requires an immediate fix.
```

### Priority 2: Serious
Color:

```
#ff920f
```

Description:

```
Requires a fix before deadline.
```

### Priority 3: Moderate
Color: default yellow

Description:

```
Should be fixed before deadline if time permits.
```

# Checklist for each GitHub org

- [ ] Create GitHub Project Board
- [ ] Set up individual product repos for audit tracking (see checklist above)

# Steps for each GitHub org

## Create Project Board

Title:

```
2025 {orgname} ADA Title II Project
```

Short Description: 

```
Accessibility remediation for the April 24, 2026 ADA Title II compliance deadline. 
```

README: 

```
This GitHub Project aggregates issues and resources for complying with the April 2024 ADA Title II rule, which requires following WCAG 2.1 Level AA guidelines for all public-facing digital assets. 

The compliance deadline for state and federal institutions with a state/local government constituent population of more than 50,000 (including UNL) is April 24, 2026.

The CDRH ADA Title II Project consists of 5 stages:

1. Planning (including inventory and prioritization of products/assets)
2. Assessment (including issue creation and prioritization)
3. Remediation (product-by-product resolution of issues and bugs)
4. Training (integrating digital accessibility into CDRH processes and individual roles and responsibilities)
5. Reporting (documentation and communication with PIs/product teams)

## Planning Documents and Resources

- [Checklist of products to be remediated](https://uofnebraska.sharepoint.com/:x:/r/sites/UNL-UniversityLibraries/DISC/cdrh/Shared%20Documents/CDRH%20Projects/Project%20Folders/Accessibility%20Audits/2025-2026%20ADA%20Title%20II%20Project/Planning%20Documents/Checklist%20of%20Products%20for%20ADA%20Title%20II%20Project.xlsx?d=w52aafefaa1a841b09bae916225c22cbb&csf=1&web=1&e=9tXWgN)
- [SharePoint folder](https://uofnebraska.sharepoint.com/:f:/r/sites/UNL-UniversityLibraries/DISC/cdrh/Shared%20Documents/CDRH%20Projects/Project%20Folders/Accessibility%20Audits/2025-2026%20ADA%20Title%20II%20Project?csf=1&web=1&e=197Dm9)
- [References and Training Materials | CDRH SharePoint](https://uofnebraska.sharepoint.com/:f:/r/sites/UNL-UniversityLibraries/DISC/cdrh/Shared%20Documents/CDRH%20Projects/Project%20Folders/Accessibility%20Audits/2025-2026%20ADA%20Title%20II%20Project/Documentation/References%20and%20Training%20Materials?csf=1&web=1&e=YNNd4o)
- [CDRH Accessibility Wiki (in progress)](https://github.com/CDRH/accessibility/wiki)

## Creating Issues

All issues should be added to the {reponame} GitHub repository. Any necessary reorganization between repositories will be done by the project lead.

### Assignee

Issues should be assigned to Erin Kay Chambers, project lead, who will reassign tasks to developers or programmers if needed.

```

**Comment by erinchambers** [comment]
Finished setup for first round of sites. More to come when I get decisions from KD+AJ and when I coordinate with GT.
